packages/sdk/src/index.ts

```diff
@@ -99,6 +100,36 @@ export default class Gomu {
     );
   }
 
+  async makeSellOrder({
+    assets,
+    amount,
+    paymentTokenAddress,
```

Comment:
Think we can squash these 2 params to just `erc20Asset`. Wanted to use `baseAssets` and `quoteAssets` to make it even more general, but @kennethtiong feels that it's not layman enough.